doc/xds-test-descriptions.md

```diff
@@ -17,6 +17,11 @@ Server should accept these arguments:
     *   When set to true it uses XdsServerCredentials with the test server for security test cases.
         In case of secure mode, port and maintenance_port should be different.
 
+In addition, when handling requests, if the initial request metadata contains the `rpc-behavior` key, it should modify its handling of the request as follows:
+
+ - If the value matches `sleep-<number>`, the server should wait the specified number of seconds before responding.
+ - If the value matches `fail-on=<hostname>`, and the specified hostname matches the server's hostname, the server should respond to the request with an error with the code `ABORTED`.
```

Comment:
Recommend to define the error message , to avoid this code being thrown by other part of the library.

Reply:
The Java server currently uses the error message `Rpc failed as per the rpc-behavior header value: <rpc-behavior value>`. I don't think it's important for this to be a specific value that the client or driver can check for, because the call stats just count the error codes.

Reply:
Using the current interop design, you are right, we don't have a way to validate the message content. I guess in future we should take this into account.

Reply:
I don't think our tests should be checking the message like that. I think it should have a "reasonable" message, but am happy with it being implementation-defined.

I would be happy if we had the error descriptions for the interop tests, but that is more about debugging and not having a test assert a particular message.